packages/communication-react/CHANGELOG.md

```diff
@@ -6,23 +6,21 @@ This log was last generated on Mon, 13 Jun 2022 18:29:25 GMT and should not be m
 
 ## [1.3.0](https://github.com/azure/communication-ui-library/tree/@azure/communication-react_v1.3.0)
 
+This stable release is mostly minor changes and bug fixes, there are no major feature introductions.
+
+A lot of bug fixes in this stable release are related to accessibility. Fixes include improved narration from screen readers in CallWithChat composite as well as improved keyboard navigation.
+
+Included as well are updates to the video streams with new features like a loading spinner in the remote stream when the remote participants connection is loading. As well we introduced the ability to update the local video streams scaling mode without destroying the current local video stream.
```

Comment:
This is just a beta feature for composites, but stable for components --- so maybe best not to mention the loading spinner here